Lib/glyphsLib/builder/custom_params.py

```diff
@@ -137,6 +138,10 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
             if UFO2FT_FILTERS_KEY not in ufo.lib.keys():
                 ufo.lib[UFO2FT_FILTERS_KEY] = []
             ufo.lib[UFO2FT_FILTERS_KEY].append(filter_struct)
+        elif name == "Replace Feature":
+            tag, repl = re.split("\s*;\s*", value, 1)
+            ufo.features.text = replace_feature(tag, repl,
+                                                ufo.features.text or "")
```

Comment:
Just to make sure that I understand, is this a master or instance parameter?

Reply:
also good question. the `set_custom_parameter` function is generic (by design) and doesn't know whether it's been called for setting font's, master's or instance's custom parameters. 

I expect this "Replace Feature" custom parameter to be useful only for instances. But I don't mind if it happens to work for masters as well.

We don't check anywhere whether a custom parameter is valid in the context of font, masters or instances.